src/common/interop/two_way_pipe_message_ipc.cpp

```diff
@@ -425,7 +425,7 @@ void TwoWayPipeMessageIPC::TwoWayPipeMessageIPCImpl::start_named_pipe_server(HAN
 
             if (token != NULL)
             {
-                int err = change_pipe_security_allow_restricted_token(connect_pipe_handle, token);
+                change_pipe_security_allow_restricted_token(connect_pipe_handle, token);
```

Comment:
Maybe log this error?

Reply:
Here I think it checks the content of connect_pipe_handle and acts accordingly.
And there is no logging going on in this file.